applications/serial_lte_modem/src/gnss/Kconfig

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SLM_GNSS
+	bool "GPS support in SLM"
+	default y
+
+config SLM_GPS
+	bool "GPS support in SLM"
+	default y if SLM_GNSS
+	select NRF9160_GPS
+
+if SLM_GPS
+
+config SLM_AGPS
+	bool "nRF Cloud A-GPS support in SLM"
+	select AGPS
+	select CLOUD_API
+	select MODEM_INFO
+	select NRF_CLOUD
+	select NRF_CLOUD_AGPS
+
+endif
+
+#config SLM_CELL_POS
+#	bool "nRF Cloud cellular postioning support in SLM"
+#	default y if SLM_GNSS
+#	select CLOUD_API
+#	select MODEM_INFO
+#	select NRF_CLOUD
+#	select NRF_CLOUD_CELL_POS
```

Comment:
Should this be removed if this is not yet supported?

Reply:
Let's keep it and I'll uncomment out this as soon as the cell_pos PR is done